### PR TITLE
inbox_for_chat_nopeconversations#149 Done

### PR DESCRIPTION
**Pull request by kamyh**

### CODE REVIEW

**Comment by kamyh** [comment]
I'm not sure how to cover this "return false". It's whene there's no pending messages.

**Comment by kamyh** [comment]
In StyleCI:

image

But in the code:

image

File: app/ChatUser.php [hunk 1]
```diff
@@ -19,8 +19,50 @@ public function sender()
...
+        return self::where(function ($query) use ($id_1, $id_2) {
+            return $query->where('reciever_id', '=', $id_1)
+                ->where('sender_id', '=', $id_2);
+        }
+        )->orWhere(function ($query) use ($id_1, $id_2) {
+            return $query->where('reciever_id', '=', $id_2)
+                ->where('sender_id', '=', $id_1);
+        }
+        )->orderBy('updated_at', 'desc')
+            ->first()->updated_at;
```
Comment (ice-blaze) [inline app/ChatUser.php]: Isn't created at a better one ?

File: app/ChatUser.php [hunk 2]
```diff
@@ -19,8 +19,50 @@ public function sender()
...
+    {
+        return self::where(function ($query) use ($id) {
+            return $query->where('reciever_id', '=', $id)
+                ->orWhere('sender_id', '=', $id);
+        }
+        )->where(function ($query) use ($id) {
+            return $query->where('sender_id', '=', $id)
+                ->orWhere('reciever_id', '=', $id);
+        }
+        )->orderBy('updated_at', 'desc');
```
Comment (ice-blaze) [inline app/ChatUser.php]: same question here ``created_at`

File: app/Http/Controllers/ChatUserController.php [hunk 3]
```diff
@@ -48,9 +72,9 @@ public function edit(ChatUser $chat)
         }
 
         return view('layouts.edit_text')->with([
-            'item'          => $chat,
```
Comment (ice-blaze) [inline app/Http/Controllers/ChatUserController.php]: won't be an issue with the PR an `.php_cs`

File: app/ChatUser.php (hunk 1 above)
Comment (ice-blaze) [inline app/ChatUser.php]: Yes but edition != last comment on the page. for example
```
James: 01.01.01 Hellow how are you ?
Nicolas: 02.01.01 Hi, I'm fine.
```
Now James see that he has wrongly spelled hello and do the change

```
Nicolas: 02.01.01 Hi, I'm fine.
James: 01.01.01 Hello how are you ?
```

I don't think it's the behaviour you expected in a chat

File: app/Http/Controllers/ChatUserController.php (hunk 3 above)
Comment (kamyh) [inline app/Http/Controllers/ChatUserController.php]: not concerning this PR

**Pull request merged**